tests/test_frontdoor.py

```diff
@@ -2032,41 +2031,6 @@ def test_on_epc_select_page_enter_manually_can_be_selected():
     assert page.has_one('h1:contains("What is the property\'s address?")')
 
 
-@unittest.mock.patch("help_to_heat.frontdoor.interface.EPCApi", MockNotFoundEPCApi)
```

Comment:
this test is no longer meaningful; the user no longer needs to click the select manual link